core/services/kraken/api/v1/routers/extension.py

```diff
@@ -0,0 +1,69 @@
+from typing import Any
+
+from commonwealth.utils.streaming import streamer
+from fastapi import APIRouter, HTTPException, status
+from fastapi.responses import StreamingResponse
+from fastapi_versioning import versioned_api_route
+
+from api.v1.models import Extension
+
+# Global Kraken control instance
+from kraken import kraken_instance
+
+# Creates Extension router
+extension_router_v1 = APIRouter(
+    prefix="/extension",
+    tags=["extension_v1"],
+    route_class=versioned_api_route(1, 0),
+    responses={status.HTTP_404_NOT_FOUND: {"description": "Not found"}},
+)
+
+# Endpoints
```

Comment:
This kind of comments, that replicate what we can see in the code just generates noise, please, try to avoid them, keep only comments that are necessary.

Reply:
Removed.